proposals/lip-0061.md

```diff
@@ -453,7 +453,7 @@ Let `h` be the height of the current tip of the chain. Commit messages in `nonGo
 1. Cleanup the data structures `nonGossipedCommits` and `gossipedCommits`:
    1. Remove any single commit message `m` from `nonGossipedCommits` and `gossipedCommits` with `m.height <=  getMaxRemovalHeight()`.
    2. For every commit message `m` in `nonGossipedCommits` or `gossipedCommits` one of the following two conditions has to hold, otherwise it is discarded.
-      * The value of `m.height` is in `{getBFTHeights().maxHeightPrecommitted - COMMIT_RANGE_STORED, …, getBFTHeights().maxHeightPrecommitted}`.
+      * The value of `m.height` is in `{getBFTHeights().maxHeightPrecommitted - COMMIT_RANGE_STORED, …, currentHeight}`, where `currentHeight` is the height of the latest block added to the blockchain.
```

Comment:
Good catch, I assume this is a leftover from some previous different validation rules. I now replaced `h` with `currentHeight` and removed the explanatory sentence that I added.

Reply:
Looks good. In fact, we could even define `currentHeight` even earlier (e.g., line 443) since it is used in step 3 of the validation. But this is optional comment and it's fine to me to merge it as is.

Reply:
I would keep it as is.